apps/docs/azure-functions/src/lib/auth.ts

```
@@ -2,50 +2,130 @@
-    // Decode JWT payload (without validation - in prod, use proper JWT validation)
-    const [, payload] = token.split(".");
-    const decoded = JSON.parse(Buffer.from(payload, "base64").toString());
-    return decoded.sub || decoded.oid || null;
-  } catch {
+  return claims?.sub || null;
+}
+
+/**
+ * Get token claims from request (validated)
+ */
+export async function getTokenClaims(
```

ESLint

[error] Multiple exports of name 'getTokenClaims'.

(import/export)